pkg/hyper/sandbox.go

```diff
@@ -190,7 +190,10 @@ func (h *Runtime) StopPodSandbox(podSandboxID string) error {
 	code, cause, err := h.client.StopPod(podSandboxID)
 	if err != nil {
 		glog.Errorf("Stop pod %s failed, code: %d, cause: %s, error: %v", podSandboxID, code, cause, err)
-		return err
+		err = h.checkpointHandler.RemoveCheckpoint(podSandboxID)
```

Comment:
should check error message and only remove checkpoint when pod has already been removed.

Also network plugin should be cleaned up before removing checkpoint.